CGPs/0023.md

```diff
@@ -10,43 +10,42 @@
 
 This governance proposal enables oracle addresses to make them able to report the value of CELO/EUR for the cEUR contract, deployed on [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md). It also proposed to send 100 CELO to each of the oracles to cover gas fees for the first couple of years and freezing StableTokenEUR (cEUR) and ExchangeEUR.
 
-Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in an status where it would be imposible to mint or burn due empty buckets that are not possible to update (assuming no oracle reports, but cEUR will be possible to move. Freezing is meant to prevent cEUR from moving before enabling the stability mechanism.
+Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in an status where it would be imposible to mint or burn due empty buckets that are not possible to update (because Mento buckets are zero and there are no oracles enable to updated), but cEUR will be possible to move. Freezing is meant to prevent cEUR from moving before enabling the stability mechanism.
 
-cEUR in circulation with no stability mechanism in place can lead to a depledge. Although the risk of cEUR moving before its full activation it's very low, this proposal will add an extra ledger of security. The only to address holding cEUR after CR3 is deployed will by the community fund (Governance Proxy) and a Celo Foundation wallet.
+cEUR in circulation with no stability mechanism in place can lead to a depledge. Although the risk of cEUR moving before its full activation it's very low, this proposal will add an extra layer of security. The only two addresses holding cEUR after CR3 is deployed will by the [Community Fund](https://celocommunityfund.org/) (Governance Proxy) and a Celo Foundation wallet.
```

Comment:
nit: I might be wrong but I think the correct term is `depeg` / `de-peg`. Depledge is something a bit different from my understanding:


> depeg
> To remove a previously instituted peg on a currency. For example, if Currency A is pegged to Currency B at a 1:1 ratio, but the central bank for Currency A decides to let it float, it is said to be depegged from Currency B. Depegging may occur if the peg is causing inflation or if the central bank is unable to sustain the peg for other reasons.

---

> depledge
> (transitive, intransitive) To revoke one's pledge of allegiance to a college fraternity.